dist/placeholder-buster.txt

```diff
@@ -51,6 +51,20 @@ dailymotion.com##div[class^="AdDiscovery"]
 # https://github.com/NanoAdblockerLab/NanoContrib/pull/16#issuecomment-473483169
 myspace.com##.container-main > ul > li:has(div[id="ad-placement"])
 
+# https://github.com/NanoAdblockerLab/NanoContrib/pull/17
+!#if env_mobile
+instagram.com##footer:style(margin-top: 25px !important)
+!#endif
+
+# https://github.com/NanoAdblockerLab/NanoContrib/pull/17
+techadvisor.*##.adsbygoogle
+techadvisor.*###topLeaderBoardHolder
+techadvisor.*###marketPlace
+techadvisor.*###bottomLeaderBoardHolder
```

Comment:
I see `techadvisor.co.uk#@#.adsbygoogle` in uBlock unbreak filter. Can you investigate?

Reply:
It's listed in Unbreak with the note "Images missing", so I guess I've got few other choices than to trust them on it.